pttrack/models.py

```diff
@@ -71,8 +71,9 @@ def make_filepath(instance, filename):
     while carry_on:
         new_filename = "%s.%s" % (User.objects.make_random_password(48),
                                   filename.split('.')[-1])
-        path = '/'.join([instance.__class__.__name__.lower(),
-                         field_name, new_filename])
+        #path = '/'.join([instance.__class__.__name__.lower(),field_name, new_filename])
+
+        path = new_filename
```

Comment:
Did we decide if we're comfortable just committing this and using the root directory of media forever?